DeploymentExperiences.md

```diff
@@ -1,8 +1,37 @@
+Link to my Django App [here](https://github.com/Nicocchi/heroku-deploy-test)
```

Comment:
Jeremy, I really appreciate that you put in all these links to make your project easy to follow and view